CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 * [`FPDF.write_html()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html) delays unescaping data so as not to confuse entity names as nested tags
 ### Changed
 * [`FPDF.table()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.table): If cell styles are provided for cells in heading rows, combine the cell style as an override with the overall heading style.
+* Update .pre-commit-config.yaml to pull in version 1.5.4
```

Comment:
```suggestion
```
This is not really something that matters for the end-users of `fpdf2` 😅

Let's try to keep this Changelog succinct

Reply:
Sure no problem, I only included that as per the checklist :)